IntelliTect.TestTools.Console.Tests/ConsoleAssertTests.cs

```diff
@@ -119,6 +119,21 @@ public void ConsoleTester_HelloWorld_DontNormalizeCRLF()
             });
         }
 
+        [TestMethod]
+        public void ConsoleTester_DoesNotStripPluses()
```

Comment:
In general the naming the unit test in this project follows the pattern:
<What is being tests>_<Test Condition>_<Expected Outcome>(). 

So for this, perhaps something like:
```suggestion
        public void ConsoleTester_OutputIncludesPluses_PlusesAreNotStripped()
```